sql/core/src/main/scala/org/apache/spark/sql/execution/metric/SQLMetrics.scala

```diff
@@ -163,6 +171,8 @@ object SQLMetrics {
         Utils.bytesToString
       } else if (metricsType == TIMING_METRIC) {
         Utils.msDurationToString
+      } else if (metricsType == NANO_TIMING_METRIC) {
+        duration => Utils.msDurationToString(duration / 100000)
```

Comment:
is this the right conversion from nanosecs to millisecs?

Reply:
Sorry...Sorry for this, change it to `1000 / 1000` as other place do for safety.